beetsplug/lyrics.py

```diff
@@ -257,8 +263,8 @@ def fetch(self, artist, title):
         html = self.fetch_url(url)
         if not html:
             return
-        lyrics = extract_text_between(html,
-                                      '"body":', '"language":')
+        lyrics = extract_text_between(html, '<p class="mxm-lyrics__content" data-reactid="135">',
+            '</p>')
```

Comment:
This extraction seems a little brittle… maybe we can find a way to avoid including the "data-reactid".